FAQ.md

```diff
@@ -677,29 +677,31 @@ O formalnościach związanych z zaliczaniem praktyk więcej [tutaj](https://foru
 
 <details>
 
-<summary>Nie zaliczyłem ćwiczeń i/lub pracowni, co dalej?</summary>
+<summary>Nie zaliczyłem ćwiczeń/pracowni/egzaminu, co dalej?</summary>
 
-> Jeśli będziesz chciał ponownie przystąpić do danego przedmiotu (bo np. jest on obowiązkowy), będziesz musiał zapłacić za jego powtarzanie. Opłata za ponowne przystąpienie do ćwiczeń/pracowni w wymiarze 30 godzin wynosi ok. 600 zł; za każde niezaliczone zajęcia pomocnicze jest naliczana osobna opłata. Suma opłat jest podzielona na 4 raty, płatne co miesiąc w ustalonym terminie. Jego przekroczenie generuje niewielkie odsetki.
+> Jeśli będziesz chciał ponownie przystąpić do danego przedmiotu (bo np. jest on obowiązkowy), będziesz musiał zapłacić za jego powtarzanie.
 >
+> W przypadku niezaliczenia ćwiczeń/pracowni, musisz zapłacić za powtarzanie ćwiczeń/pracowni oraz wykładu (i ponownie uczęszczać na ćwiczenia/pracownię).
+> W przypadku niezaliczenia egzaminu, musisz zapłacić tylko za powtarzanie wykładu, ćwiczenia/pracownia pozostają zaliczone i nie musisz uczęszczać na nie ponownie.
+> 
+> Opłata za powtarzanie przedmiotu wynosi 20 zł za godzinę zajęć dla osób rozpoczynających studia w roku 2022 lub później. Dla osób które rozpoczęły studia wcześniej - 6 zł za godzinę.
+>
+> Przykład 1: XYZ rozpoczął studia w 2020 roku. Nie zaliczył ćwiczeń z algebry. Musi zapłacić za powtarzanie ćwiczeń oraz wykładu. Ćwiczenia trwają 2h tygodniowo, wykład 3h. XYZ musi zapłacić (2h + 3h) * 15 * 6zł/h = 450zł (15 to liczba tygodni w semestrze).
+>
+> Przykład 2: XYZ rozpoczął studia w 2022 roku. Nie zdał egzaminu z AISDu. Musi zapłacić tylko za powtarzanie wykładu (4h tygodniowo). Musi on zapłacić 4h * 15 * 20zł/h = 1200zł.
 
 </details>
 
 <details>
 
-<summary>Nie zaliczyłem egzaminu, co dalej?</summary>
+<summary>Jak zapisać się na przedmiot ponownie? Gdzie i do kiedy muszę zapłacić?</summary>
 
-> ~~**Od roku akademickiego 2020/2021:** Niezaliczenie egzaminu = niezaliczenie wykładu. Za powtarzanie wykładu należy zapłacić (około 6 złotych za godzinę wykładu).~~
+> Na przedmiot zapisujesz się tak samo jak zwykle, przez System Zapisów. W przypadku zaliczenia ćwiczeń/pracowni należy się zapisać tylko na wykład.
 >
-> **Od roku akademickiego 2022/2023:** Jeśli zaliczyłeś zajęcia pomocnicze (ćwiczenia/pracownie), przy następnym podejściu do tego przedmiotu nie musisz zaliczać ich od nowa. Za powtarzanie wykładu cena wynosi 20 zł za godzinę (jeśli rozpocząłeś studia przed 2022 rokiem, dalej obowiązuje Cię stara stawka 6 zł za godzinę).
+> Po rozpoczęciu semestru w systemie USOS pojawią się nierozliczone należności wraz z terminem do kiedy należy je uregulować (można je zapłacić w 4 ratach lub od razu w całości):
```

Comment:
Może rozbijmy to jednak na dwa punkty? Pierwszy byłby "Jak zapisać się ponownie na przedmiot", i to by było krótkie. Co do drugiego, to może coś w stylu "jak wyglądają formalności"? Piszemy tu też o jakichś papierach, to może być istotne a może uciec jak nie nazwiemy tego punktu odpowiednio.

Reply:
Ma sens, zaraz przerobię